### PR TITLE
MGMT-2423 Add ability to search clusters by name, id or base domain

### PR DESCRIPTION
**Pull request by jtomasek**
Adjust toolbar to match OCM toolbar layout

### CODE REVIEW

File: src/selectors/clusters.tsx [hunk 1]
```diff
@@ -31,9 +31,11 @@ const clusterToClusterTableRow = (cluster: Cluster): IRow => {
     cells: [
       {
         title: (
-          <Link key={name} to={`${routeBasePath}/clusters/${id}`} id={`cluster-link-${name}`}>
-            {name}
-          </Link>
+          <>
```
Comment (mareklibra) [inline src/selectors/clusters.tsx:34]: Is this needed?

File: src/components/clusters/ClustersTable.tsx [hunk 2]
```diff
@@ -115,7 +115,15 @@ const ClustersTable: React.FC<ClustersTableProps> = ({ rows, deleteCluster }) =>
 
   const rowFilter = React.useCallback(
     (row: IRow) => {
-      if (searchName && !(row.props.name || '').toLowerCase().includes(searchName.toLowerCase())) {
+      const searchableProps: string[] = [
```
Comment (mareklibra) [inline src/components/clusters/ClustersTable.tsx:118]: good idea

File: src/components/clusters/ClustersTable.tsx [hunk 3]
```diff
@@ -115,7 +115,15 @@ const ClustersTable: React.FC<ClustersTableProps> = ({ rows, deleteCluster }) =>
...
+        row.props.name,
+        row.props.id,
+        row.props.baseDnsDomain,
+      ].map((prop) => (prop || '').toLowerCase());
+      if (
+        searchString &&
+        !searchableProps.filter((prop) => prop.includes(searchString.toLowerCase())).length
```
Comment (mareklibra) [inline src/components/clusters/ClustersTable.tsx:125]: nit: This can be simplified by using `Array.find()`

**Review by mareklibra — CHANGES_REQUESTED** [review]

File: src/selectors/clusters.tsx (hunk 1 above)
Comment (jtomasek) [reply src/selectors/clusters.tsx:34]: good catch, thanks!

File: src/components/clusters/ClustersTable.tsx (hunk 3 above)
Comment (jtomasek) [reply src/components/clusters/ClustersTable.tsx]: done

**Review by mareklibra — APPROVED** [review]

**Pull request merged**